cmd/gke-exec-auth-plugin/main.go

```diff
@@ -7,6 +7,8 @@ import (
 
 	"cloud.google.com/go/compute/metadata"
 
+	"github.com/gofrs/flock"
```

Comment:
group this with the library above (remove empty line 9)

Reply:
Done.